index.js

```diff
@@ -62,7 +63,28 @@ function initConfig(projectConfig) {
       minorVersion: process.env.GAE_MINOR_VERSION
     }
   };
-  var config = extend(true, {}, require('./config.js'), projectConfig, envConfig);
+
+  var envSetConfig = {};
+  if (process.env.hasOwnProperty('GCLOUD_TRACE_CONFIG')) {
+    try {
+      envSetConfig = require(path.resolve(process.env.GCLOUD_TRACE_CONFIG));
+    } catch (error) {
+
+      var logger = common.logger({
+        level: common.logger.LEVELS[2],
+        tag: '@google-cloud/trace-agent',
+      });
+
+      logger.error('The environment variable GCLOUD_TRACE_CONFIG was set,' +
```

Comment:
My personal preference would be to just require the value of `GCLOUD_TRACE_CONFIG` directly if it is present. This would reduce the chance that a user believes their environmental configuration is in effect when it really failed to load in case they miss the log (especially since many configuration may not be immediately observable such as trace buffer size, flush delay, or sampling rate). WDYT? If this seems reasonable I'm happy to update at landing time.